health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/CreateCampaignComponents/CampaignNameInfo.js

```
@@ -0,0 +1,53 @@
+    );
+  
+    if(infoLoading){
+      return <Loader page={true} variant={"PageLoader"} />;
+    }
+
+  return (
+    <AlertCard
+            label={t("CONSOLE_NAMING_CONVENTION")}
+            style={{ width: "100%", maxWidth: "unset" }}
+            additionalElements={[
+              <div className="campaign-name-parent-container">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)